build.sbt

```diff
@@ -590,16 +590,25 @@ lazy val serialversionRemoverPlugin =
 lazy val serialversionRemoverPluginSettings = Seq(
   Compile / scalacOptions ++= (
       if (scalaVersion.value.startsWith("3."))
-        Seq("-Xplugin:" + (serialversionRemoverPlugin / Compile / Keys.`package`).value.getAbsolutePath.toString)
+        Seq("-Xplugin:" + (serialversionRemoverPlugin / Compile / Keys.`package`).value.getAbsolutePath)
```

Comment:
This was noted by IntelliJ as unnecessary as `getAbsolutePath` returns a string.